src/components/NewsCard/NewsCard.tsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import cx from 'classnames';
+
+import { ImageElement } from 'Image';
+import FormattedDate from 'FormattedDate';
+import Link from 'Link';
+
+type NewsCardProps = {
+  author: string;
+  className?: string;
+  date: string;
+  image: {
+    alt: string;
```

Comment:
I'm becoming less of a fan of these nested props as it makes them less visible. PageHeaderCompact has a very flat structure and it's instantly more readable.

I also think over time it would be good to refactor other components which have nested props.

Reply:
good point - I think you're right, have updated